project/plugins.sbt

```diff
@@ -50,12 +52,12 @@ addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.8.0")
   * better ivy alternative for dependency resolution
   * @see https://github.com/alexarchambault/coursier
   */
-addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.2")
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-RC13")
```

Comment:
@bastiion any reason to roll this back?

Reply:
Hi, i think this might have happened by accident. if dependency resolution works fine, there would be no reason to roll this back.